packages/sanity/src/core/validation/validators/dateValidator.ts

```diff
@@ -12,6 +12,7 @@ const isoDate =
 // eslint-disable-next-line no-warning-comments
 // TODO (eventually): move these to schema type package
 interface DateTimeOptions {
+  // @todo accept object representing `Intl.DateTimeFormatOptions`? how do we localize?
```

Comment:
Do we have a shortcut story for this? 👁️

Reply:
Not yet, will create one :) 
It's not really a must-have, but I thought it might be interesting to explore it.